package.json

```diff
@@ -5,9 +5,9 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "server:development": "nodemon src/server/bootstrap/bootstrap",
+    "server:development": "NODE_ENV=development nodemon src/server/bootstrap/bootstrap",
```

Comment:
Useless, default env will be development, we must get environment only from `config.get('env')`.